OOPConsoleApp/Program.cs

```diff
@@ -0,0 +1,273 @@
+﻿using OOPConsoleApp.Interface;
+
+namespace OOPConsoleApp
+{
+    internal class Program
+    {
+        private static void Main()
+        {
+            Exercise1();
+            Exercise2();
+            Exercise3();
+            Exercise4();
+            Exercise5();
+
+            void Exercise1()
+            {
+                Console.WriteLine("Exercise 1.");
+
+                var person = new Person("Fleming", "Lise", 33);
+
+                Console.WriteLine($"{person.GetSurname()} " +
+                    $"{person.GetName()}, " +
+                    $"{person.GetAge()}");
+            }
+
+            void Exercise2()
+            {
+                Console.WriteLine("Exercise 2.");
+
+                var student = new Student("Koval", "Irina", 19);
+                student.SetStudentAbility("musical ear");
+                student.SetStudentAssesment(5);
+
+                Console.WriteLine($"Student - {student.PersonSurname}" +
+                    $" {student.PersonName}," +
+                    $" {student.PersonAge} years old.");
+                Console.WriteLine($"Ability: {student.StudentAbility}. Assesment: {student.StudentAssesment}.");
+
+                var teacher = new Teacher("Ivanov", "Ivan", 41);
+                teacher.SetTeacherTemperament("sanguine");
+                teacher.SetTeacherSubject("Mathematicks");
+
+                Console.WriteLine($"Teacher - {teacher.PersonSurname} " +
+                    $"{teacher.PersonName}, " +
+                    $"{teacher.PersonAge} years old. ");
+                Console.WriteLine($"Temperament: {teacher.TeacherTemperament}. Subject: {teacher.TeacherSubject}.");
+            }
+            void Exercise3()
+            {
+                Console.WriteLine("Exercise 3.");
+
+                var personSecond = new Person("Datchin", "Valery", 43);
+
+                Console.WriteLine($"{personSecond.GetSurname()} {personSecond.GetName()}," +
+                $" {personSecond.GetAge()} y.o.");
+            }
+            void Exercise4()
+            {
+                Console.WriteLine("Exercise 4.");
+
+                var myPerson = new Person("Fleming", "Lise", 33);
+                myPerson.Print();
+
+                var myStudent = new Student("Koval", "Irina", 19);
+                myStudent.Print();
+
+                var myTeacher = new Teacher("Ivanov", "Ivan", 41);
+                myTeacher.Print();
+            }
+
+            void Exercise5()
+            {
+                Console.WriteLine("Exercise 5.");
+
+                var classroom = new Classroom("Pupkin", "Vasiliy", 20);
+
+                classroom.AddPerson("Kirkorov", "Petya", 46);
+                classroom.AddPerson("Vasilyeva", "Iren", 18);
+
+                classroom.Print();
+
+                classroom.RemovePerson("Pupkin", "Vasiliy", 20);
+                classroom.RemovePerson("Kirkorov", "Petya", 46);
```

Comment:
Why do you need all this data to delete? Define one identifier, take an example from the project